cross/libtheora/Makefile

```diff
@@ -0,0 +1,18 @@
+PKG_NAME = libtheora
+PKG_VERS = 1.1.1
+PKG_EXT = tar.bz2
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://downloads.xiph.org/releases/theora/
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = cross/libogg cross/libvorbis
+
+HOMEPAGE = http://www.theora.org/
+COMMENT  = Theora is a free and open video compression format from the Xiph.org Foundation.
+LICENSE  =
```

Comment:
http://www.theora.org/faq/#14, BSD-style. Don't know if there's a direct link to the COPYING file somewhere that could be used otherwise.